src/main/java/com/JSONpostroute/functions/orders/OrderFunctions.java

```diff
@@ -1,4 +1,18 @@
 package com.JSONpostroute.functions.orders;
 
+import com.JSONpostroute.models.RequestJSON;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
 public class OrderFunctions {
+    public static int totalOrders(List<RequestJSON> orderList){
+        int total_orders = 0;
+        for (int i = 0; i < orderList.size(); i++) {
+            int orderAddend = orderList.get(i).getQuantity();
+            total_orders += orderAddend;
```

Comment:
I think you don't need this var orderAddend because this code it's short and easy to read and if you iterate 10 times you will create it 10 times in the memory, iterate 100000 will create... and the performance will down.

_**total_orders = orderList.get(i).getQuantity();**_

Reply:
In this case you should create this var outside to the "for" if you want to separate the code (...getQuantity).

Reply:
I'm good with both solutions, 1) create the var outside or 2) remove the var.